### ISSUE TITLE
Rotation stops working as expected

### ISSUE THREAD

**Issue by Abrokenfish** [ArchiDog1998/RotationSolver#517]
ok so, i have a similar issue with the Standard step going off early but, im more unsure why it just stops working completely then when standard procks up again it automatically is used but no other ability is used i have tried on multiple jobs and now it is the same for them for example when im playing as MCH it'll only use the 3 button combo and nothing else at all?

i've tried this on the default rotation, Updated version of DNC and Vex's rotation etc but i have the same issue for all jobs where it'll do some abilities then just stop working.

**Comment by ArchiDog1998** [comment]
Yeah, DNC has a bug and this will be fixed very soon. I just tested MCH rotation in Default, and Inco's rotations, it is not only using 3 combos, but also the others. I'll update a new version of RS, it may be fixed in the next version. If it is still not working fine, please tell me.
